src/tests/controllers/TextStyleController.test.ts

```diff
@@ -1,19 +1,26 @@
 import { TextStyleController } from '../../controllers/TextStyleController';
 import MockEditorAPI from '../__mocks__/MockEditorAPI';
 
-let mockedTextProperties: TextStyleController;
+describe('TextProperties', () => {
+    let mockedTextProperties: TextStyleController;
 
-beforeEach(() => {
-    mockedTextProperties = new TextStyleController(MockEditorAPI);
-    jest.spyOn(mockedTextProperties, 'setTextStyleProperties');
-});
+    beforeEach(() => {
+        mockedTextProperties = new TextStyleController(MockEditorAPI);
+        jest.spyOn(mockedTextProperties, 'setTextStyleProperties');
+        jest.spyOn(mockedTextProperties, 'clearTextStyleProperties');
+    });
 
-afterAll(() => {
-    jest.restoreAllMocks();
-});
-describe('TextProperties', () => {
-    it('Should call all of the TextStyle Functions of EditorAPI successfully', () => {
+    afterAll(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('Should call setTextStyleProperties of EditorAPI successfully', () => {
         mockedTextProperties.setTextStyleProperties({ FONT_SIZE: { value: 34 } });
         expect(mockedTextProperties.setTextStyleProperties).toHaveBeenCalledTimes(1);
     });
+
+    it('Should call cleanTextStyleProperties of EditorAPI successfully', () => {
+        mockedTextProperties.clearTextStyleProperties();
+        expect(mockedTextProperties.clearTextStyleProperties).toHaveBeenCalledTimes(1);
```

Comment:
We should probably align to something like `mockedSDK.editorAPI.clearTextStyleProperties` https://github.com/chili-publish/editor-sdk/pull/247/files#diff-a1691cce13cbe768f2d037bc9b482a986fb438e0251a3c12129cc2fcb96ef664R26 as otherwise this is a dummy test.